src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -55,6 +55,9 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
```

Comment:
Can you use HTML hidden attribute here? https://www.w3schools.com/TAGs/att_global_hidden.asp

Reply:
Alternatively, you can make this logic in your model since you reuse it

Reply:
Isn't it already hidden based on a condition? It looks like using the html hidden attribute would do the same thing in terms of being able to the condition by looking at the js source code. I think moving this logic internally would be a prettier solution, but I'm not sure how to do this right now. I'll come back to this after the best friend feature.